### PR TITLE
check kubelet rootdir if equal kubelet rootdir mounted in csi

### PR DESCRIPTION
**Pull request by zwwhdls**

### CODE REVIEW

**Review by Hexilee — APPROVED** [review]
LGTM

File: pkg/juicefs/juicefs.go [hunk 1]
```diff
@@ -358,6 +364,37 @@ func (j *juicefs) GetJfsVolUUID(ctx context.Context, name string) (string, error
...
+	if kubeletDir != dirs[0] {
+		return fmt.Errorf("target kubelet rootdir %s is not equal csi mounted kubelet root-dir %s", dirs[0], kubeletDir)
+	}
+	return nil
+}
+
+func (j *juicefs) validOptions(volumeId string, options []string) error {
+	for _, option := range options {
+		if option == "writeback" {
+			klog.Warningf("writeback is not suitable in CSI, please donot use it. volumeId: %s", volumeId)
```
Comment (timfeirg) [inline pkg/juicefs/juicefs.go:392]:
```suggestion
			klog.Warningf("writeback is not suitable in CSI, please do not use it. volumeId: %s", volumeId)
```

File: pkg/juicefs/juicefs_test.go [hunk 2]
```diff
@@ -446,7 +447,7 @@ func Test_juicefs_JfsMount(t *testing.T) {
 		})
 		Convey("ce format error", func() {
 			volumeId := "test-volume-id"
-			targetPath := "/target"
+			targetPath := "/var/lib/kubelet/pods/a019aa39-cfa9-42fd-9b26-1a4fd796212d/volumes/kubernetes.io~csi/pvc-090cf941-0dcd-4ddc-8099-b86dd6caa5eb/mount"
```
Comment (timfeirg) [inline pkg/juicefs/juicefs_test.go:450]: extract into a variable?

Comment (zwwhdls) [reply pkg/juicefs/juicefs_test.go:450]: fine in unit test

**Pull request merged**